packages/components/src/components/notification-banner/notification-banner.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * @license
+ * Scale https://github.com/telekom/scale
+ *
+ * Copyright (c) 2021 Egor Kirpichev and contributors, Deutsche Telekom AG
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import {
+  Component,
+  h,
+  Host,
+  Prop,
+  Element,
+  Method,
+  State,
+} from '@stencil/core';
+import classNames from 'classnames';
+import statusNote from '../../utils/status-note';
+
+const TIMEOUT = 3000;
+
+@Component({
+  tag: 'scale-notification-banner',
+  styleUrl: 'notification-banner.css',
+  shadow: true,
+})
+export class NotificationMessage {
+  @Element() hostElement: HTMLElement;
+
+  @Prop() variant?: 'informational' | 'success' | 'warning' | 'error' =
+    'informational';
+  @Prop() dismissible?: boolean = false;
+  @Prop({ reflect: true }) opened: boolean;
+  @Prop() timeout?: boolean | number = false;
```

Comment:
this is confusing - can we think of a better name? like auto-hide? and auto-hide-duration? so we don't have two types for one prop and the naming is self explanatory.